src/components/App/App.js

```diff
@@ -1,21 +1,40 @@
 import React, { Component } from 'react';
 import Header from '../Header/Header.js';
 import Container from '../Container/Container.js';
+import { Route, Switch } from 'react-router-dom';
 
 export default class App extends Component {
   constructor() {
     super()
     this.state = {
       movies: [],
-      currentMovie: ''
+      currentMovie: null
     }
   }
 
+  updateCurrentMovie = (id) => (
+    this.setState({ currentMovie: id })
+  )
+
+
   render() {
     return (
       <div>
         <Header />
-        <Container movies={this.state.movies}/>
+        <Switch>
+          <Route
+            path="/:title"
+            render={() => (
+            <Container movies={this.state.movies} currentMovie={this.state.currentMovie} updateCurrentMovie={this.updateCurrentMovie}/>
+          )}
```

Comment:
I think the spacing here needs to be in one more indent. The `)}` on line 29 should be in line with the inner `render`.